opencti-platform/opencti-front/src/private/components/common/entreprise_edition/EnterpriseEditionButton.tsx

```diff
@@ -33,7 +33,7 @@ const EnterpriseEditionButton = ({
   const {
     settings: { id: settingsId },
   } = useAuth();
-  const isAdmin = useGranted([SETTINGS]);
+  const isAdmin = useIsAdmin();
```

Comment:
quick refactoring on the fly, 
I did not make use this new hook in the final version of my task, but I though it was worth keeping anyway !